src/04_plot_testresults.R

```diff
@@ -50,7 +49,7 @@ main <- function(){
   null_dist <- mode_rank %>% 
     specify(response = rank, explanatory = mmode) %>% 
     hypothesize(null = "independence") %>% 
-    generate(reps = 10000, type = "bootstrap") %>% 
+    generate(reps = 10000, type = "permute") %>% 
```

Comment:
@sedv8808: Are you sure about changing this?  This will mean that we are using different methods for generating our CI's (they use `bootstrap`, and changing them to `permute` will cause an error) than our p-values.  Maybe this doesn't matter, but since the change will affect our CI and P-values (although not our conclusion), if it is okay to change, I think we should address it in the comments.